src/components/atoms/Ranges/DefaultRange/Range.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import { Input, Label, StyledWrapper } from './Range.styled';
+import { IProps } from './Range.model';
+
+const Range = ({ className, type }: IProps): JSX.Element => {
+  const [rangeValue, setRangeValue] = useState(1);
+
+  return (
+    <StyledWrapper className={className}>
+      <Label htmlFor="range" id="range" aria-label="Range">
+        <Input
+          onChange={(e) => setRangeValue(+e.target.value as number)}
+          type={type}
```

Comment:
We have only one type and this is the same for each cases. I think we dont need this props

Reply:
ok